base/REPLCompletions.jl

```diff
@@ -152,14 +152,17 @@ end
 # Determines whether method_complete should be tried. It should only be done if
 # the string endswiths ',' or '(' when disregarding whitespace_chars
 function should_method_complete(s::AbstractString)
+    method_complete = false
     for c in reverse(s)
         if c in [',', '(']
-            return true
+            method_complete = true
+            break
         elseif !(c in whitespace_chars)
-            return false
+            method_complete = false
+            break
         end
     end
-    false
+    method_complete
 end
```

Comment:
The one return point vs three is largely a matter of taste, and I am rather indifferent in this case. It seems like a bit silly transformation just to test one more line of code, but you already did the work. So, I'm fine with keeping it.